### DISCUSSION TITLE
Toast Visual Enhancements

### DISCUSSION THREAD

**Discussion by gsgou** [CommunityToolkit/Maui#1047] [New Feature Discussions]
**Summary**
A couple (hopefully) small changes/additions that would add a lot of flexibility and functionality to Toast.

**API Changes**
Define text alignment: In ToastOptions, remove bool IsRtl and add TextAlignOption TextAlign where
TextAlignOption { Left, Center, Right}.
Define width: In ToastOptions, add bool FullWidth where True renders the Toast from edge to edge, false renders it the length of the text + padding
Define a float: In ToastOptions, add int FloatMargin that will float the toast given number of pixels above the anchor.

**Intended Use Case**
![image](https://user-images.githubusercontent.com/911534/222160477-9e99b97b-355d-4857-9699-596cf79a14b5.png)

**Comment by VladislavAntonyuk** [comment]
not sure if all of these suggestions can be done. we are limited by google implementation of snackbar and toast and can only provide that functionality.

**Comment by CliffAgius** [comment]
Closed and answered
